### PR TITLE
Add addedOn and unit to tentant model

### PR DESCRIPTION
**Pull request by cphpitts**
### What issue is this solving?
https://github.com/codeforpdx/dwellingly-app/issues/254
Add 'addedOn' and 'unit' properties to the Tenant model

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? Yes
Any new dependencies to install? No
Any special requirements to test? No

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

File: resources/tenants.py [hunk 1]
```diff
@@ -21,6 +22,8 @@ class Tenants(Resource):
     parser.add_argument('phone',type=str,required=True,help="This field cannot be blank.")
     parser.add_argument('propertyID',required=False,help="This field can be provided at a later time.")
     parser.add_argument('staffIDs',action='append',required=False,help="This field can be provided at a later time.")
+    parser.add_argument('unit',action='append',required=False,help="This field can be provided at a later time.")
+    parser.add_argument('addedOn',type=datetime,required=True,help="This field cannot be blank.")
```
Comment (NickSchimek) [inline resources/tenants.py]: It's my understanding that the addedOn column would be the date the object is created? Is this date coming from the frontend or can we create it in the backend when we save to the db?

**Review by NickSchimek — COMMENTED** [review]
Thanks for the PR @cphpitts - I have one concern regarding the required addOn parameter. Please see the below comment. Otherwise everything looks good. There seems to be some failing tests after these changes, I didn't take a look at them, however, they will need to be fixed before we can merge. Let me know if you have any questions.

File: resources/tenants.py (hunk 1 above)
Comment (NickSchimek) [reply resources/tenants.py]: It looks like addedOn parameter is not even being used in the endpoint. This line can simply be removed as it is not a required argument.

**Review by NickSchimek — CHANGES_REQUESTED** [review]

File: models/tenant.py [hunk 2]
```diff
@@ -36,7 +40,9 @@ def json(self):
             'phone': self.phone,
             'propertyID': self.propertyID,
             'propertyName': self.property.name if self.property else None,
-            'staff': [user.json() for user in self.staff] if self.staff else None
+            'staff': [user.json() for user in self.staff] if self.staff else None,
+            'unit': self.unit,
+            'addedOn': self.addedOn
```
Comment (NickSchimek) [inline models/tenant.py]: This is causing the majority of the failures. A date object is not serializable. It needs to be converted to a string.
`self.addedOn.strftime("%m/%d/%Y, %H:%M:%S")`

Comment (cphpitts) [reply models/tenant.py]: This is resolved

**Comment by cphpitts** [comment]
@NickSchimek I believe I have resolved the issues. Let me know if there's anymore issues.

File: models/tenant.py (hunk 2 above)
Comment (NickSchimek) [reply models/tenant.py]: The strftime method was placed in the initializer. It should be here in the json method. We want to store the date in the db as a date object, and when we send to front end it should be the string representation.

After making these changes and running tests locally you should then see 2 failing tests. Those also will need to be resolved. We can pair on it if you like.

**Comment by cphpitts** [comment]
All errors have been resolved.

**Review by NickSchimek — DISMISSED** [review]
Thanks @cphpitts . Looks good!

**Review by NickSchimek — APPROVED** [review]

**Pull request merged**